R/sas_dates.R

```diff
@@ -37,9 +42,11 @@ sas_numeric_to_date <- function(date_num, datetime_num, time_num, tz = "") {
     if (!all(mask_na_match)) {
       stop("The same values are not NA for both `date_num` and `time_num`")
     }
-    ret <- as.POSIXct(86400 * date_num + time_num, origin = "1960-01-01", tz = tz)
-  } else if (has_datetime) {
-    ret <- as.POSIXct(datetime_num, origin = "1960-01-01", tz = tz)
+    datetime_num <- 86400 * date_num + time_num
+    has_datetime <- TRUE
+  }
+  if (has_datetime) {
+    ret <- as.POSIXct(datetime_num, origin = "1960-01-01", tz = "UTC")
```

Comment:
Is the `tz = "UTC"` here intentional instead of `tz = tz` ? Then the input value of `tz` won't be used? It sounds like in #583 that was initially your approach, then you clarified that you want to keep existing code working.

Reply:
Thanks for the review. You're right, I did intend to use `tz = tz`.